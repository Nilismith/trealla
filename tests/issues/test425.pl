:- initialization(main).
:- use_module(library(lists)).

main :-
	catch(length(_,1000000),E,(write(hello),nl)), write(E), nl.
