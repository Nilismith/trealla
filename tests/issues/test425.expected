hello
error(resource_error(memory),length_rundown/2)
